src/main/java/Roody.java

```diff
@@ -152,13 +163,25 @@ public static void main(String[] args){
             // else, repeat
             } else if (inputs[0].equals("list")) {
                 roody.printList();
+            } else if (inputs[0].equals("delete")) {
+                if (inputs.length == 2) {                    
+                    roody.delete(inputs[1]);
+                } else {
+                    new RoodyException("Please enter a index number to be deleted");
```

Comment:
Maybe can try using switch statements instead of if-statements here to have a neater code 😄 
the method split() can be helpful in determining "delete", "list" or "mark" cases etc!

Reply:
Thanks for this, made it smoother.